dev/gallery.tsx

```diff
@@ -95,6 +95,9 @@ export function Gallery() {
     );
 
     const [isMobile, setIsMobile] = useState(params.get("mobile") === "true");
+    const [showTooltips, setShowTooltips] = useState(
+        params.get("tooltips") === "true",
```

Comment:
Can this be shortened to `useState(params.get("tooltips"))`? or `!!useState(params.get("tooltips"))`?

Reply:
I didn't see that "true" is a string. You can ignore my suggestion!